uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/build.js

```diff
@@ -4,9 +4,10 @@
   wrapShim: true,
   modules: [
     {name: 'admin-dashboard'},
+    {name: 'absence-tab'},
     {name: 'my-leave'},
     {name: 'manager-leave'},
-    {name: 'absence-tab'}
+    {name: 'waiting-approval-notification'}
```

Comment:
I think maybe something like `manager-notification-badge` (or `manager-`?) would be better, so we don't tie it to a specific status. In case the logic behind the badge number should change, the name would still be relevant